test_cases/test_dojo.py

```diff
@@ -97,23 +100,136 @@ def test_print_unallocated_with_filename_unspecified(self):
 		self.the_dojo.add_person("Marie", "marie@marie.com","Fellow", "y")
 		self.the_dojo.add_person("Pete","pete@pete","Staff")
 		output = self.the_dojo.print_unallocated()
-		expected_output = ("\n\n\tUNALLOCATED PEOPLE\n" + "*" * 50 
-			+ "\n\tMarie\tmarie@marie.com\tFellow\ty \n" 
-			+ "\n\tPete\tpete@pete.com\tStaff ")
+		expected_output = ("\n\n LIST OF ALL UNALLOCATED STAFF AND FELLOWS\n" 
+			+ "*" * 50 
+			+ "\n\tMarie\tmarie@marie.com\tFellow\tOFFICE SPACE\n" 
+			+ "\n\tPete\tpete@pete.com\tStaff\tOFFICE SPACE\n"
+			+ "\n\tMarie\tmarie@marie.com\tFellow\tLIVING SPACE\n" )
 		self.assertEqual(output, expected_output)
 			
 	def test_print_unallocated_with_filename_specified(self):
 		self.the_dojo.add_person("Marie", "marie@marie.com","Fellow", "y")
 		self.the_dojo.add_person("Pete","pete@pete","Staff")
-		output = self.the_dojo.print_unallocated("file1")
-		expected_output = "\nData has been successfully saved to file2.txt\n"
+		output = self.the_dojo.print_unallocated("file2")
+		expected_output = "Saving unallocations list to file...\
+		\nData has been successfully saved to file2.txt\n"
 		self.assertEqual(output, expected_output)
 
-	def test_if_print_unallocated_filepath_exists(self):
-		self.the_dojo.print_unallocated("file2")
-		self.assertTrue(os.path.isfile("file2.txt"))
-		os.remove("file2.txt")
+	def test_reallocate_person_returns_error_if_room_doesnot_exist(self):
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo.add_person("Pete", "pete@pete", "staff")
+		result =self.the_dojo.reallocate_person("pete@pete", "Green")
+		self.assertEqual(result, "The OFFICE Green doesnot exist!")
 
+	def test_reallocate_person_successfully_reallocates_person(self):
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo.add_person("Pete", "pete@pete", "staff")
+		self.the_dojo.create_room("office", "Green")
+		room_occupancy = [room.occupants for room in self.the_dojo.all_offices\
```

Comment:
Since you know that office "green" is second on the list, simply say:
`room_occupancy = self.the_dojo.all_offices[1].occupants` It will spare you easy bugs